web-gateway/conf/messages

```diff
@@ -87,3 +87,10 @@ invalid.units=Please enter a valid unit
 createUserInstructions=Here you can create a new user who can participate in and create auctions.
 itemMessage=Click on an item to view/edit its details.
 editItemInstructions=Here you can edit details of your auction item. If the auction has not started, you can edit any detail including description, minimum bid increment, auction duration, etc. However, if the auction is on-going, only the item description can be edited.
+instruction.createItem=Here you can create new items to auction.
+instruction.placeBid=Here you can place a bid for item under auction.
+instruction.seller=Here you can start the auction of the item.
+instruction.bidder.auctionStarted=The auction has started for this item.
+instruction.seller.auctionStarted=This item is under auction. Now other users can start bidding for this item.
+instruction.auctionEnded=The auction has ended for this item.
+instruction.auctionCancelled=The auction has been cancelled for this item.
```

Comment:
I think the properties names are inconsistent with each other and with existing values. 
I like the `instructions.` prefix you introduced and I'd suggest we stick to it. I would redo the two following:
```
instruction.auctionStarted.bidder
instruction.auctionStarted.seller
```